packages/qwik-city/runtime/src/types.ts

```diff
@@ -377,14 +377,10 @@ export type Editable<T> = {
 
 type UnionKeys<T> = T extends T ? keyof T : never;
 type StrictUnionHelper<T, TAll> = T extends any
-  ? T & Partial<Record<Exclude<UnionKeys<TAll>, keyof T>, never>>
+  ? T & Partial<Record<Exclude<UnionKeys<TAll>, keyof T>, undefined>>
```

Comment:
This change from `never` to `undefined` might not be strictly necessary - I was being overly cautious, but it seems to build either way and not cause errors on property access. I can revert this line if desired.